### PR TITLE
Hcalnano autonano

### PR DESCRIPTION
**Pull request by vlimant**
#### PR description:

harmonise how hcal DPG nano are configured and tested (#43961) 

FYI: @DryRun

#### PR validation:

nano matrix succeded

a backport "might" be included in #44428

### CODE REVIEW

**Comment by vlimant** [comment]
enable nano

**Comment by vlimant** [comment]
type hcal

**Comment by vlimant** [comment]
please test

**Comment by vlimant** [comment]
please test

**Comment by vlimant** [comment]
+1

**Comment by vlimant** [comment]
can we please have the remaining signatures, or comments for this trivial PR ? @cms-sw/operations-l2 @cms-sw/pdmv-l2 @cms-sw/upgrade-l2.
Is there a better way to bring such trivial changes forward in quicker way before some conflicts emerge @cms-sw/orp-l2

File: Configuration/PyReleaseValidation/python/relval_nano.py [hunk 1]
```diff
@@ -154,18 +157,24 @@ def subnext(self):
...
 
 steps['muDPGNANO_data13.0']=merge([{'-s' : 'RAW2DIGI,NANO:@MUDPG',
-                                   '--conditions':'auto:run3_data',
-                                   '-n' : '100',
-                                   '--era' : 'Run3',
-                                   '--datatier':'NANOAOD',
-                                   '--eventcontent':'NANOAOD'}])
+                                    '-n' : '100',},
+                                   steps['NANO_data13.0']])
+
```
Comment (AdrianoDee) [inline Configuration/PyReleaseValidation/python/relval_nano.py:162]: I imagine this, while untreated to the PR, is just that you took the chance to implement the `merge` + `steps['NANO_data13.0']` syntax also for `@MUDPG` auto NANO.

**Comment by AdrianoDee** [comment]
+pdmv

**Comment by antoniovilela** [comment]
ping @cms-sw/upgrade-l2 
Thanks.

**Comment by antoniovilela** [comment]
> can we please have the remaining signatures, or comments for this trivial PR ? @cms-sw/operations-l2 @cms-sw/pdmv-l2 @cms-sw/upgrade-l2. Is there a better way to bring such trivial changes forward in quicker way before some conflicts emerge @cms-sw/orp-l2

Please bring PRs that are taking time to get all signatures during the weekly ORP meeting, where we can also discuss the requested backports.
We had indeed questions concerning backports in the meeting yesterday.
Thanks.

File: Configuration/PyReleaseValidation/python/relval_nano.py (hunk 1 above)
Comment (vlimant) [reply Configuration/PyReleaseValidation/python/relval_nano.py:162]: yes, sorry indeed.

**Comment by vlimant** [comment]
I am away this week, I'll try to connect to the ORP next week if time permits

**Comment by srimanob** [comment]
+Upgrade

**Comment by antoniovilela** [comment]
+1

File: Configuration/PyReleaseValidation/python/relval_nano.py (hunk 1 above)
Comment (AdrianoDee) [reply Configuration/PyReleaseValidation/python/relval_nano.py:162]: No worries of course

**Pull request merged**